src/hooks/useFeedbackDisclosure.ts

```diff
@@ -0,0 +1,66 @@
+import { useDisclosure } from "@chakra-ui/react"
+import { useLocation } from "react-router-dom"
+
+import { LOCAL_STORAGE_KEYS } from "constants/localStorage"
+
+import { useLoginContext } from "contexts/LoginContext"
+
+import { isEditPageUrl, isSpecialPagesUrl } from "utils/pages"
+
+import { useLocalStorage } from "./useLocalStorage"
+
+type LastSeenFeedbackTime = number
```

Comment:
curious here, what type benefit are we getting here ah? this seems to attempt to use the `Brand {__kind: UserId}` flow

Reply:
for _nominal_ type systems (ts is structural), they wouldn't be mutually assignable. however, the value here is just so that devs know what this is actually intended to be. Tbh i can do the `brand` pattern too, it's not a big deal